chia/util/initial-config.yaml

```diff
@@ -423,6 +426,9 @@ full_node:
   network_overrides: *network_overrides
   selected_network: *selected_network
 
+  trusted_peers:
+    0ThisisanexampleNodeID7ff9d60f1c3fa270c213c0ad0cb89c01274634a7c3cb7: Does_not_matter
```

Comment:
I think this could be explained in a more user-friendly way. Instead of having a dummy entry, maybe you could add a comment describing what one can put in this list.

I take it we're not adding our own local wallet's peer ID in here automatically then